arches/app/media/js/utils/map-popup-provider.js

```diff
@@ -40,6 +42,34 @@ define(['arches',
             return features;
         },
 
+        /**
+         * This method enables custom logic for how the feature in the popup should be handled and/or mutated en route to the mapFilter.
+         * @param popupFeatureObject - the javascript object of the feature and its associated contexts (e.g. mapCard).
+         * @required @method mapCard.filterByFeatureGeom()
+         * @required send argument: @param feature - a geojson feature object
+         * @optional send argument: @param resourceid
+         */
+        sendFeatureToMapFilter: function(popupFeatureObject)
+        {
+            let feature = popupFeatureObject.geometries()[0].geom.features[0];
+            // Note that polygons with very high vertex-counts can benefit from simplification.
```

Comment:
Perhaps this strategy should be included in the documentation rather than in the code base?

Reply:
fix in latest, created https://github.com/archesproject/arches-docs/issues/417